payments-core/src/main/java/com/stripe/android/networking/StripeApiRepository.kt

```diff
@@ -1172,41 +1170,6 @@ class StripeApiRepository @JvmOverloads internal constructor(
         }
     }
 
-    /**
-     * Retrieves the ConsumerSession if the given email is associated with a Link account.
-     */
-    override suspend fun lookupConsumerSession(
```

Comment:
any reason why we want to extract this as a separate one?

Reply:
This PR just moves this endpoint, but all consumer endpoints will also be moved on following PRs.

This endpoints handle Link user sessions (login, signup, 2FA, etc) that are now going to be used in Connections.